packages/jest-snapshot/src/common.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+const common = Object.create(null);
```

Comment:
don't use `Object.create` for things you export, it is only necessary when you use objects as map without a finite set of keys that you know upfront :) Also, it should go after the require.